pkg/capture/populate.go

```diff
@@ -74,6 +74,9 @@ func populate(p *capturetypes.GPPacket, pkt capture.Packet) error {
 			}
 
 			if protocol == capturetypes.TCP {
+				if len(srcPacket) < ipv4.HeaderLen+13 {
```

Comment:
Another nice outcome of this issue: For invalid TCP packets (could certainly be crafted that way in a malicious scenario) the whole thing would crash without this check.